client/src/components/Comments/Comment.jsx

```diff
@@ -1,30 +1,31 @@
-import React from "react";
-import PropTypes from "prop-types";
+import React from 'react';
+import PropTypes from 'prop-types';
+import avatar from 'assets/logo.png';
 
-import avatar from "../../assets/logo.png";
+const Comment = ({ comment }) => {
+	const { person, content } = comment
 
-const Comment = function Comment(props) {
-    const { comment } = props;
-    return (
-        <li className="comment-main-level">
-            <div className="comment-avatar">
-                <img src={avatar} alt="avatar" />
-            </div>
-            <div className="comment-box">
-                <div className="comment-head">
-                    <h6 className="comment-name">
-                        <a href={"/profile/" + comment.person.id}>
-                            {comment.person.alias || "אנונימי"}{" "}
-                        </a>
-                    </h6>
-                </div>
-                <div className="comment-content"> {comment.content} </div>
-            </div>
-        </li>
-    );
+	return (
+		<li className="comment-main-level">
+			<div className="comment-avatar">
+				<img src={avatar} alt="avatar" />
+			</div>
+			<div className="comment-box">
+				<div className="comment-head">
+					<h6 className="comment-name">
+						<a href={'/profile/' + person.id}>
```

Comment:
Maybe we can remove this href, we don't use the '/profile'  page in this version